crates/net/network/src/test_utils/testnet.rs

```diff
@@ -106,7 +106,7 @@ where
     /// # Panics
     /// If the index is out of bounds.
     pub fn remove_peer(&mut self, index: usize) -> Peer<C, Pool> {
-        self.peers.remove(index)
+        self.peers.swap_remove(index)
```

Comment:
this we can't do because this messes with the ordering

Reply:
oh ok, didn't know peers ordering was important, will revert